src/ContentRepository90/Rules/FusionFlowQueryContextRector.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Neos\Rector\ContentRepository90\Rules;
+
+use Neos\Rector\Core\FusionProcessing\EelExpressionTransformer;
+use Neos\Rector\Core\FusionProcessing\FusionRectorInterface;
+use Neos\Rector\Utility\CodeSampleLoader;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+class FusionFlowQueryContextRector implements FusionRectorInterface
+{
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return CodeSampleLoader::fromFile('Fusion: Add comments for q(node).context({targetDimensions|currentDateTime|removedContentShown|inaccessibleContentShown: ...})', __CLASS__);
+    }
+
+    public function refactorFileContent(string $fileContent): string
+    {
+        return EelExpressionTransformer::parse($fileContent)
+            ->addCommentsIfRegexMatches(
+                '/context\(\s*\{(.*)[\'"](targetDimensions|currentDateTime|removedContentShown|inaccessibleContentShown)[\'"](.*)\}\s*\)/',
```

Comment:
thanks for taking care .. i think this must be now one of the most expensive regexes ... but for now good enough ^^

Reply:
Yeah... Idk. It's just applied to single lines, so should be fine.